doc/cli-directive.md

```diff
@@ -79,6 +79,12 @@ For groupName, operationName, parameterName, typeName, propertyName, usually you
     - old: 'old_value'
     - new: 'new_value'
     - isRegex: true | false
+- split-operation-names
+  - split operation into multiple operations with given names
+  - value format:
+    - opName1
+    - opName2
+    - ...
```

Comment:
it will be helpful to also have some sample below

Reply:
Addded. Thanks.